libs/androidlib/src/mill/androidlib/AndroidAppKotlinModule.scala

```diff
@@ -131,24 +129,15 @@ trait AndroidAppKotlinModule extends AndroidKotlinModule with AndroidAppModule {
     // FIXME: avoid hardcoded version
     def uiToolingVersion: String = "1.7.6"
 
-    /*
-     * A relaxed resolution policy, as this module runs the tests
-     * on the host machine, outside of Android, but it still needs
-     * the android ui dependencies to work, so production wise this is
-     * low risk.
-     * @return
-     */
-    override def resolutionParams: Task[ResolutionParams] = Task.Anon {
-      val params = super.resolutionParams()
-      relaxedDependencyReconciliation(params)
-    }
-
-    private val relaxedDependencyReconciliation: ResolutionParams => ResolutionParams =
-      _.withReconciliation(Seq(ModuleMatchers.all -> Reconciliation.Relaxed))
-
     override def generatedSources: T[Seq[PathRef]] = Task { Seq.empty[PathRef] }
 
-    override def mandatoryMvnDeps: T[Seq[Dep]] = super.mandatoryMvnDeps() ++
+    override def resolvedMvnDeps: T[Seq[PathRef]] = Task {
+      defaultResolver().classpath(
```

Comment:
@alexarchambault for some reason I couldn't get the same dependency set with the millResolver used in the inherited `resolvedMvnDeps` . I think all of the dependencies that were missing had a "runtime" configuration, but I'm not too sure .

Reply:
That's odd… there ought to be very little difference between the two. The only difference is that `millResolver` knows about the modules defined in the build, while `defaultResolver` doesn't.

Reply:
I think it's due to the default variant matchers in resolvedMvnDeps in JavaModule afterall, I'm still experimenting with the settings